src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Test/ScenarioTests/IaasVm/ItemTests.ps1

```diff
@@ -213,6 +213,13 @@ function Test-AzureVMFullRestore
 		
 		Set-AzureRmRecoveryServicesVaultContext -Vault $vault;
 
+		Assert-ThrowsContains { Restore-AzureRmRecoveryServicesBackupItem `
```

Comment:
Can you add a positive test for this parameter?

Reply:
This feature is not enabled by default, so setting up the test artefacts for the positive case is quite difficult. Will take this up at a later time when the feature is enabled by default.